test/CMakeLists.txt

```diff
@@ -143,10 +143,10 @@ endif()
 
 if(ENABLE_NODE_BINDINGS)
```

Comment:
@allierowan this was dropping its test logs in the source directory. also i named it `.log` to match the rest of the tests